### ISSUE TITLE
Strange URLs in my log

### ISSUE THREAD

**Issue by guymass** [WWBN/AVideo#8570]
I keep seeing URLs like this:

No Cache (/program/103492/2/RaisingTheFrequency/%D7%A9%D7%99%D7%93%D7%95%D7%A8-1/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC6832721/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC6832721/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC6832721/%22) FROM: 52.70.240.171 Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_10_1) AppleWebKit/600.2.5 (KHTML, like Gecko) Version/8.0.2 Safari/600.2.5 (Amazonbot/0.1; +https://developer.amazon.com/support/amazonbot) SCRIPT_NAME: /view/modeYoutube.php

This is more clear:

/program/103492/2/Onelove/%D7%A9%D7%99%D7%93%D7%95%D7%A8-1/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/researchreview.in/index.php/rr/article/view/78/119/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC6832721/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC8427918/%22https:/www.ncbi.nlm.nih.gov/pmc/articles/PMC6832721/%22

This string is because it is in hebrew font - %D7%A9%D7%99%D7%93%D7%95%D7%A8

constantly repeating on strange posts names of articles which don't appear to be videos im not sure. 

Are these normal URLs?

**Comment by guymass** [comment]
Also now caught this error in log, not sure where it should go. 
![Screenshot 5784-02-10 at 3 09 32](https://github.com/WWBN/AVideo/assets/58882903/a3ea3202-fe69-4e23-9165-244da45bda9d)

**Comment by guymass** [comment]
![Screenshot 5784-02-10 at 3 17 37](https://github.com/WWBN/AVideo/assets/58882903/c5172890-5c1c-4ba3-af88-f96c279861bb)

**Comment by guymass** [comment]
Just another large screenshot just to show the amount of requests and strange urls, I am not sure but this is slowing down the site for sure, my log file is getting filled up very fast.
![Screenshot 5784-02-10 at 3 26 18](https://github.com/WWBN/AVideo/assets/58882903/3308c452-bba8-494e-800d-ed559685f196)

**Comment by guymass** [comment]
Another strange output in log, not sure if its an issue:
![Screenshot 5784-02-10 at 3 35 07](https://github.com/WWBN/AVideo/assets/58882903/9851c139-add8-475d-b217-4e58eaec2655)

**Comment by guymass** [comment]
Here is my log, can you please tell me if those URL's are normal and why my log is filled so fast, I have errors and the site is currently working very badly, it getting very slow idk really. I am uploading my log file in hopes someone will be able to understand it better then I.

[avideo.log](https://github.com/WWBN/AVideo/files/13245534/avideo.log)

**Comment by DanielnetoDotCom** [comment]
the URLs are OK

I suspect your cachesInDB table has wrong indexes

you can recreate it like this

run this https://github.com/WWBN/AVideo/blob/master/install/recreateCacheDB.php

```
cd /var/www/html/AVideo/install
php recreateCacheDB.php
```

**Comment by guymass** [comment]
I keep doing that all the time and it keeps having this issue so what else can I do/check?

**Comment by DanielnetoDotCom** [comment]
What is the issue ?

**Comment by guymass** [comment]
OK I think my main issue was maybe related to my network encoders. I currently managed to change my configuration to give me much better loading time for my network encoders. I am not sure if its related, but could be that while they were working in the old configuration I had very long time delay which might have cause overall the site to work slowly. Currently I have very low latency accessing the encoders and everything seems to work better. I will keep testing over the next few days and see if this improves my performance. 

Current ping is now very low compare to previous configuration:

![Screenshot 5784-02-20 at 0 41 54](https://github.com/WWBN/AVideo/assets/58882903/e36ef5f4-cda2-4c01-b037-69508aa52edc)

**Comment by guymass** [comment]
After few hours of running fine it starts again to hang and I see this repeating in the log:

![Screenshot 5784-02-21 at 13 48 26](https://github.com/WWBN/AVideo/assets/58882903/5b0b5bc1-9c62-4ddd-9f37-2789e777c20e)

**Comment by guymass** [comment]
This Show as well:

![Screenshot 5784-02-21 at 13 55 20](https://github.com/WWBN/AVideo/assets/58882903/7c2ea1a2-fa4c-4cba-b765-db37c01e562f)

**Comment by guymass** [comment]
And I also see this error on my network encoders server:
![Screenshot 5784-02-21 at 14 12 01](https://github.com/WWBN/AVideo/assets/58882903/5b6607e2-75ca-4741-9871-6c5265388493)

**Comment by guymass** [comment]
About two hour since I ran " php recreateCacheDB.php " and again site is hanging:

![Screenshot 5784-02-21 at 16 40 29](https://github.com/WWBN/AVideo/assets/58882903/1f1fe339-ddb7-4f7b-ac69-9ea47fd3bda4)

Simply trying to refresh the videos page get hanged and all I see in the log is those repeating urls which IDK what they are for.

![Screenshot 5784-02-21 at 16 43 25](https://github.com/WWBN/AVideo/assets/58882903/ac6d882d-1913-4f53-9e66-f3e4a734a7f1)
![Screenshot 5784-02-21 at 16 44 46](https://github.com/WWBN/AVideo/assets/58882903/8f9822c8-e119-4ed0-8d43-d4c50a15596b)
![Screenshot 5784-02-21 at 16 44 20](https://github.com/WWBN/AVideo/assets/58882903/a093f4ec-6ddd-44a8-a1c1-1b4f644e426b)

**Comment by guymass** [comment]
![Screenshot 5784-02-21 at 16 48 20](https://github.com/WWBN/AVideo/assets/58882903/d6673b43-6b9f-4473-b8dc-d84cda8f200b)

**Comment by guymass** [comment]
This is from my network encoders server and I think there is a connection, maybe:

![Screenshot 5784-02-21 at 17 08 34](https://github.com/WWBN/AVideo/assets/58882903/c77f3602-fcb3-4bb3-8d4d-7cca4825ae4e)

**Comment by guymass** [comment]
For now I noticed I had low timeout value on my networks encoders server, I must have missed it, was set to 60 and not 600.

I now set all values for both Timeout and ProxyTimeout parameters in the apache and php config INI files to 7200 and finally it seems to load the encoders iframe properlly fast. I will keep testing just to make sure.

**Comment by guymass** [comment]
I still see some errors on the network encoders server and it hangs when I try to import a video from YT so I need to refresh to see if it ran the script of not.

![Screenshot 5784-02-21 at 17 25 08](https://github.com/WWBN/AVideo/assets/58882903/bd10b933-f2fa-4299-b50c-0796ca55f029)

![Screenshot 5784-02-21 at 17 25 22](https://github.com/WWBN/AVideo/assets/58882903/506c6efb-c7a4-4b7f-9209-9066140e9416)

**Comment by guymass** [comment]
If I refresh and quickly go to the share YT URL and run it, it will allow me eventually to get my video, but if I let the page rest for few seconds its like it is loosing its connection and it will no longer run the php script. IDK yet what is causing this.
![Screenshot 5784-02-21 at 17 37 10](https://github.com/WWBN/AVideo/assets/58882903/f7a0a073-7c3e-4d5c-958f-b5b3501c64bd)

**Comment by guymass** [comment]
More errors, now I see something regarding my network encoders configuration:
![Screenshot 5784-02-22 at 11 31 20](https://github.com/WWBN/AVideo/assets/58882903/5cab1034-d88d-413d-af68-b63372a824a6)

**Comment by DanielnetoDotCom** [comment]
your streamer cannot reach your encoder network

it is something in your local network

try to use the hosts file to speed up the connection between servers.

**Comment by guymass** [comment]
Ok thinks seem to have been working better. I tried using the hosts file it didn't make any difference. my issue is related to the performance of my Network encoders  loading time and the communication between them and the site. I have improved much the overall ping rate, however because my encoders run over a reverse proxy I guess this is the best I can get on that tunneling connection. I suppose that having a proper encoder running on a public IP would be much faster and more efficient. So for now I am closing this issue as there is not much that can be done IMO. Thanks for all the support.
